src/components/Todo/Item.jsx

```diff
@@ -1,13 +1,14 @@
 
 /* eslint-disable react/prop-types */
 function Item( props ) {
-  const { todoItem } = props
+  const { todoItem, onRemoveTodo } = props
   
   return (
-    <div>
+    <div className='todo-item'>
       <label>
         <input type="checkbox" />
```

Comment:
not sure how I didn't catch this earlier, but I didn't see any mention of checkboxes in previous assignments. if you're looking to do a mass delete functionality at some point, then great.